server/libs/skills/wd/insertPicture/insertPicture.js

```diff
@@ -0,0 +1,31 @@
+const WordSkill = require("../common/wdSkill");
+
+module.exports = class InsertPicture extends WordSkill {
+    /**
+     * 
+     * @param {*} skillParams 
+     */
+    getFileNameFinal(skillParams) {
+        try {
+            let paramValueObj = skillParams.paramsObj;
+            let imgName = paramValueObj["imageFileName"];
+            return Promise.resolve({ attrValue: [imgName, imgName.slice(0, imgName.lastIndexOf('.'))] });
+        } catch (error) {
+            return Promise.reject(error);
+        }
+    }
+
+    htmlFileUpload(inputFile, dependantDropzoneModel) {
```

Comment:
front end function
can have some naming convention signifying that it's a front end function
option 1: fn name can have a prefix "fe" => this fn name will change to "feHTMLFileUpload"